lib/logpipe.h

```diff
@@ -213,6 +213,7 @@ struct _LogPipe
   GlobalConfig *cfg;
   LogExprNode *expr_node;
   LogPipe *pipe_next;
+  StatsCounterItem *stats_discarded;
```

Comment:
I would rename it to `discarded_messages`, to stay consistent with other counters' names.